docs/admin/access.rst

```diff
@@ -212,35 +231,18 @@ keep managing your permissions site-wide for these groups too, choose an
 appropriate value for the :guilabel:`Project selection` (e.g.
 :guilabel:`All projects` or :guilabel:`All public projects`).
 
-Per-project permission management
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can set your projects to `Protected` or `Private`, and
-:ref:`manage users <manage-acl>` per-project in the Weblate interface.
-
-By default this prevents Weblate from granting access provided by
-`Users` and `Viewers` :ref:`default groups <default-groups>` due to these groups’
-own configuration. This doesn't prevent you from granting permissions to those
-projects site-wide by altering default groups, creating a new one, or creating
-additional custom settings for individual component as described bellow.
-
-One of the main benefits of managing permissions through the Weblate
-user interface is that you can delegate it to other users without giving them
-superuser privilege. In order to do so, add them to the `Administration`
-group of the project.
-
 Custom permissions for languages, components or projects
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can create your own dedicated groups to manage permissions for distinct
-objects such as languages, components and projects. Although these groups can
-only grant additional privileges, you are not able to revoke any permission
-granted by site-wide or per-project groups by adding another custom group.
+objects such as languages, components, and projects. Although these groups can
+only grant additional privileges, you can’t revoke any permission granted
+by site-wide or per-project groups by adding another custom group.
 
 **Example:**
 
   If you want (for whatever reason) to allow translation to a
-  specific language (lets say `Czech`) only to a closed set of reliable users
+  specific language (lets say `Czech`) only to a closed set of reliable translators
   while keeping translations to other languages public, you will have to:
 
   1. Remove the permission to translate to the `Czech` from all the users. In the
```

Comment:
remove "to the" from "to the `Czech`"

Reply:
Yes, can you make a suggestion?

Reply:
Somehow the ability to do so goes away if things have been edited before, or it has something to do with review.

Reply:
Done

Reply:
solved